part-spec/power.json

```diff
@@ -164,6 +164,550 @@
+                },
+                "currentLimit": {
+                    "description": "maximum sustained output current threshold beyond which the output starts drooping",
+                    "comment": "units of amps",
+                    "$ref": "definitions.json#/unit"
+                },
+                "rampTime": {
+                    "description": "time for output voltage to go from 10% vout nominal to 90% vout nominal for fixed turn-on time part",
+                    "comment": "units of seconds",
+                    "$ref": "definitions.json#/conditionalProperty"
+                },
+                "reverseCurrentBlocking": {
+                    "description": "whether load switch has capability to block current flow from vout to vin",
+                    "type": "boolean"
+                },
+                "onResistance": {
+                    "description": "nominal DC resistance of load switch when enabled",
+                    "comment": "units of ohms",
+                    "$ref": "definitions.json#/unit"
+                },
+                "package": {
+                    "description": "component's package size and description",
+                    "$ref": "definitions.json#/package"
+                },
+                "pins": {
+                    "description": "array of pin objects with associated properties",
+                    "type": "array",
+                    "items": {
+                        "$ref": "definitions.json#/pinSpec"
+                    }
+                }
+            }
+        },
+        {
+            "title": "battery charger",
+            "type": "object",
+            "required": [
+                "componentID"
+            ],
+            "properties": {
+                "componentID": {
+                    "description": "common component identifying information, such as mpn",
+                    "$ref": "definitions.json#/componentID"
+                },
+                "converterType": {
+                    "description": "switching voltage regulator type",
+                    "examples": [
+                        "buck",
+                        "boost",
+                        "buck-boost",
+                        "other"
+                    ],
+                    "type": "string"
+                },
+                "chargerTopology": {
+                    "description": "type of battery charger topology (Narrow VDC vs Hybrid Power Boost)",
+                    "enum": [
+                        "nvdc",
+                        "hpb"
+                    ],
+                    "type": "string"
+                },
+                "batteryConfig": {
+                    "description": "battery serial configurations supported by charger",
+                    "enum": [
+                        "1S",
+                        "2S",
+                        "3S",
+                        "4S"
+                    ],
```

Comment:
later when we're reviewing the properties, instead of "1S", "2S",.. we can change the property to reflect number of cells in series and have a number here.